package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "yarn",
   "installationMethod": "unknown",
-  "version": "1.23.0-atlassian.3",
+  "version": "1.23.0-atlassian.5",
```

Comment:
What was v4?

Reply:
Ah, yeah no bump on the previous change https://github.com/atlassian-forks/yarn/pull/8/files. All good 👍

Reply:
The current version in our repo is v4

Reply:
Someone must have forgotten to push the updated version number for 4

Reply:
That someone would probably have been me hehe

Reply:
From memory I fixed yarn add behaviour in 4